modules/tinymce/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Direct invalid child text nodes of list elements will be wrapped in list item elements. #TINY-4818
 
 ### Fixed
+- onSetup function would not run when defining custom group toolbar button. #TINY-9496
```

Comment:
Maybe something like this looks a bit weird to have a api reference thing as the first word.

```suggestion
- The `onSetup` api function would not run when defining custom group toolbar button. #TINY-9496
```